calicoctl.py

```diff
@@ -174,10 +174,13 @@ def container_add(container_name, ip):
         print "IP address is already assigned in pool %s " % pool
         sys.exit(1)
 
-    # Actually configure the netns.  Use eth1 since eth0 is the docker bridge.
+    # Actually configure the netns. Default to eth1 since eth0 is the
+    # docker bridge.
+    if interface is None:
```

Comment:
I think this would be clearer for both devs and users if we used docopt to set the default.